src/notification/email_sender.py

```diff
@@ -151,15 +151,18 @@ def generate_email_content(self) -> str:
                             if (
                                 not self.report_config.hide_filters
                                 and department != "single_department"
-                            ):
-                                blocks.append(f"**{department}**")
+                            ):                            
```

Comment:
```suggestion
                            ):
```
Atenção aos espaços em branco extras ao final da linha.

Sugiro fortemente usar o linter black no VSCode, rodar F1, "Format document" e salvar novamente. Ele resolve essas questões automaticamente.